src/index.js

```diff
@@ -0,0 +1,19 @@
+// eslint-disable-next-line no-unused-vars
+import _, { } from 'lodash';
+import * as method from './modules/interactive.js';
+import UpdateUi from './modules/mainSec.js';
+import './style.css';
+import { refresh } from './modules/selectors.js';
+
```

Comment:
- [ ] Hello, Your Project is working fine, but you did not follow the requirements as stated in your project requirements, ([See Project requirement](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m1_list_structure.md)). This is because you need to set an array of some simple *to do tasks* (array of objects) with each object containing three keys - description \[`string`\], completed \[`bool`\], index: \[`number`\]
- [ ] Please follow the requirements.